config.example.py

```diff
@@ -15,24 +15,31 @@
 
 SPARKPOST_KEY = ""
 
+
 class GOOGLE_CAL:
+    CAL_ID = ""
+    CAL_API_KEY = ""
+
+    # optional - only needed if private data needs to be accessed
```

Comment:
Im confused if this is still needed cause right now the cal announce function doesn't reference these keys